packages/core/android/src/test/kotlin/com/datadog/reactnative/DdLogsTest.kt

```diff
@@ -327,7 +327,7 @@ internal class DdLogsTest {
         verifyZeroInteractions(mockLogger)
         val throwableCaptor = argumentCaptor<Throwable>()
         verify(mockPromise, times(8)).reject(throwableCaptor.capture())
-        assertThat(throwableCaptor.firstValue).hasFieldEqualTo("detailMessage", "Log sent before SDK init")
+        assertThat(throwableCaptor.firstValue.message!!).isEqualTo("DD_INTERNAL_LOG_SENT_BEFORE_SDK_INIT")
```

Comment:
should be working also without `!!`, because `assertThat` accepts null values.
```suggestion
        assertThat(throwableCaptor.firstValue.message).isEqualTo("DD_INTERNAL_LOG_SENT_BEFORE_SDK_INIT")
```